stdlib/strings.wat

```diff
@@ -0,0 +1,8 @@
+(module
```

Comment:
I see – you did make this file. This could import the memory operations and export the string operations.